x-pack/plugins/security_solution/common/endpoint/service/response_actions/constants.ts

```diff
@@ -52,6 +52,27 @@ export const CONSOLE_RESPONSE_ACTION_COMMANDS = [
 
 export type ConsoleResponseActionCommands = typeof CONSOLE_RESPONSE_ACTION_COMMANDS[number];
 
+export type ResponseConsoleRbacControls =
+  | 'writeHostIsolation'
+  | 'writeProcessOperations'
+  | 'writeFileOperations'
+  | 'writeExecuteOperations';
+
+/**
+ * maps the console command to the RBAC control that is required to access it via console
+ */
+export const commandToRBACMap = new Map<ConsoleResponseActionCommands, ResponseConsoleRbacControls>(
```

Comment:
any chance you can consolidate the many Maps/Arrays we have in different place to all use this? Example: 

https://github.com/elastic/kibana/blob/e0bc286a758032f9f3127a0d06566e6721e52eca/x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts#L67-L92

Also, I don't think its best to use a Mutable `Map` for a global const. can we just use a regular object and `Object.freeze()` it?

Reply:
changed the const to be an object instead a40406782fb84e102b0470db7912ba893e3655ec

Reply:
United the maps that were there in that file f782864193289083905011787920dc04ebd8afcf. 


There's one more such map using response action commands on the [server](https://github.com/ashokaditya/kibana/blob/b3807ba0b9b0fdf32cc4423b9bd894e132387e98/x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.ts#L200) that we can add to it as well. Let me know if I should do that too. I presume I'll move this new consolidated map to `security_solution/common/endpoint/service/response_actions`?